protobuf/lib/src/protobuf/protobuf_enum.dart

```diff
@@ -37,9 +37,16 @@ class ProtobufEnum {
   /// Creates a new constant [ProtobufEnum] using [value] and [name].
   const ProtobufEnum(this.value, this.name);
 
-  /// Creates a Map for all of the [ProtobufEnum]s in [byIndex], mapping each
-  /// [ProtobufEnum]'s [value] to the [ProtobufEnum].
-  static Map<int, T> initByValue<T extends ProtobufEnum>(List<T> byIndex) {
```

Comment:
To avoid making this a breaking change, you can just keep this function under this name (even if it's unused by generated code from new generator, package may still use it)